keras/layers/recurrent.py

```diff
@@ -1468,6 +1534,7 @@ def __init__(self, units,
                  go_backwards=False,
                  stateful=False,
                  unroll=False,
+                 reset_after=True,
```

Comment:
This default value will break loading old model files.

Reply:
The default value of GRUCell is `reset_after=False`

Reply:
Both should be the same, `False`. This would be a breaking change (besides the fact that both values must be consistent).

Reply:
Sorry a silly typo. I did the changes too quickly. I've been flying the whole day and try to fix it later today.

Reply:
It caused failed test:
```
tests/keras/engine/test_topology.py::test_preprocess_weights_for_loading[layer0] FAILED
```